mesh/executor_test.go

```diff
@@ -345,9 +354,12 @@ func TestExecutor_ExecuteOptimistic(t *testing.T) {
 		}
 		expBlock.Initialize()
 		te.mcs.EXPECT().UpdateCache(gomock.Any(), expBlock.LayerIndex, expBlock.ID(), gomock.Any(), gomock.Any()).DoAndReturn(
-			func(_ context.Context, _ types.LayerID, _ types.BlockID, gotE []types.TransactionWithResult, gotI []types.Transaction) error {
+			func(_ context.Context, _ types.LayerID, gotB types.BlockID, gotE []types.TransactionWithResult, gotI []types.Transaction) error {
```

Comment:
gotB, gotE, gotI is terrible. if original names are taken can you write a full word or acceptable shorthand e.g gotBid/gotBlock, gotResults, gotTxs